src/Chirp.Web/Pages/_ViewImports.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿@using Chirp
+@namespace Chirp.Pages

```

Comment:
I presume that the namespace is missing web such that `Chirp.Web.Pages` ? :)

Reply:
This is code copied from the template and seems to work fine as when i comment it out it breaks the login button so I wouldn't worry about it